blathers: add to cockroachdb/cockroach repo

otan opened the pull request:
This should live on the cockroachdb/cockroach repo so it is easier to
edit and modify.

This has it's own `go.mod`, so it should not affect other main packages.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/61987)
<!-- Reviewable:end -->

otan committed:
blathers-bot: add to cockroachdb/cockroach repo

This should live on the cockroachdb/cockroach repo so it is easier to
edit and modify.

This has it's own `go.mod`, so it should not affect other main packages.

Release note: None

otan commented:
OMFG SUBMODULES
